classes/various/extComplex.sc

```diff
@@ -0,0 +1,26 @@
+/*
+This file is part of MathLib, a Quark for SuperCollider:
+
+<https://github.com/supercollider-quarks/MathLib>
+
+You should have received a copy of the GNU General Public License v3 along with
+this distribution. If not, see:
+
+<https://www.gnu.org/licenses/gpl-3.0.txt>.
+*/
+
+
+//---------------------------------------------------------------------
+//	MathLib: Some mathematical extensions to SuperCollider.
+//
+// 	Extension: Complex
+//
+//---------------------------------------------------------------------
+
+
+/* J Anderson, 2019 */
++ Complex {
```

Comment:
This should go into the common library. Could you make a pull request there?

Reply:
@telephon, yep, you're right!

Given that I'm _really_ looking to solve https://github.com/supercollider-quarks/MathLib/issues/20, and I'm hoping to simplify local roll-out for dev version of ATK, how do you feel about:

1. A PR refactoring uses in `Polynomial` from `Complex.sqrt` to `Complex.pow(0.5)`
2. Submitting `Complex.sqrt` addition to the common library

This route would mean that https://github.com/supercollider-quarks/MathLib/issues/20 is quickly solved before then common library solution becomes available.

Reply:
That's good!